### ISSUE TITLE
Some minor possible improvements

### ISSUE THREAD

**Issue by tizzen33** [glorang/NUPlayer#1]
First of all, thanks a lot for creating this awesome app. It works fine and incredibly fast on Proximus' Android TV decoder (v7). 

A few improvements that would make the experience even better:
- When I press the 'stop' button, the episode stops but the screen stays black until I use the 'back' button. 
- It's not possible to rewind or fast forward with the buttons on the remote. It does work if I select the rewind or fast forward icons on the screen.
- When I pause and resume an episode, the overlay stays visible until the 'back' button is pressed.

That's it.

Thanks again
tizzen

**Comment by glorang** [comment]
Thanks for your feedback, much appreciated. The video player controls were indeed one of the things that still needed improvement.

I've released v1.1.6 in the beta channel on Google Play, it should become available within the next 45 minutes or so.
It should fix at least the "Stop" issue and improve the rewind / fast forward (a lot), but possibly not yet with the rewind / forward buttons on your remote. 

I'll need to check how to reproduce, neither Chromecast and TADAAM remote have separate forward/rewind/play/stop buttons, they all stick to D-PAD only.

Lastly, the overlay issue seems a bug in the Leanback library, I've implemented some code that should fix it but doesn't seem to work here so far. Might need some more investigation.
